app/models/volume_mapping.rb

```diff
@@ -1,9 +1,60 @@
 class VolumeMapping < ApplicationRecord
+  include ProviderObjectMixin
+  include SupportsFeatureMixin
+  include CustomActionsMixin
+
```

Comment:
Alright now lets add `include NewWithTypeStiMixin` to this and host initiators (this way inventory refresh passing `:type => ""` to `new` will create the proper subclass